terraform/base/route53.tf

```diff
@@ -5,87 +5,180 @@ resource "aws_route53_zone" "mozilla-releng" {
-resource "aws_route53_record" "heroku-tooltool-cname" {
+resource "aws_route53_record" "heroku-clobberer-cname-prod" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "clobberer.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["saitama-70467.herokussl.com"]
+}
+resource "aws_route53_record" "heroku-mapper-cname-prod" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "mapper.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["mapper.mozilla-releng.net.herokudns.com"]
+}
+resource "aws_route53_record" "heroku-tooltool-cname-prod" {
     zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
     name = "tooltool.mozilla-releng.net"
     type = "CNAME"
     ttl = "180"
     records = ["kochi-11433.herokussl.com"]
 }
+resource "aws_route53_record" "heroku-treestatus-cname-prod" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "treestatus.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["treestatus.mozilla-releng.net.herokudns.com"]
+}
 
-resource "aws_route53_record" "heroku-tooltool-staging-cname" {
+
+###############################
+## Heroku staging app cnames ##
+###############################
+
+resource "aws_route53_record" "heroku-archiver-cname-stage" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "archiver.staging.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["archiver.staging.mozilla-releng.net.herokudns.com"]
+}
+resource "aws_route53_record" "heroku-clobberer-cname-stage" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "clobberer.staging.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["saitama-70467.herokussl.com"]
+}
+resource "aws_route53_record" "heroku-mapper-cname-stage" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "mapper.staging.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["mapper.staging.mozilla-releng.net.herokudns.com"]
+}
+resource "aws_route53_record" "heroku-treestatus-cname-stage" {
+    zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
+    name = "treestatus.staging.mozilla-releng.net"
+    type = "CNAME"
+    ttl = "180"
+    records = ["nagasaki-25852.herokussl.com"]
+}
+resource "aws_route53_record" "heroku-tooltool-cname-stage" {
     zone_id = "${aws_route53_zone.mozilla-releng.zone_id}"
     name = "tooltool.staging.mozilla-releng.net"
     type = "CNAME"
     ttl = "180"
     records = ["shizuoka-60622.herokussl.com"]
 }
 
-# Treestatus app cname uses non-sni ssl due to old versions of python
-# used during the build process
-# See bug 1380177
-resource "aws_route53_record" "heroku-treestatus-cname" {
+
+#########################################
+## Heroku Shipit production app cnames ##
+#########################################
+
+resource "aws_route53_record" "heroku-dashboard-shipit-cname-prod" {
```

Comment:
``dashboard.shipit.mozilla-releng.net`` can be removed since it is not used anymore